src/screens/MainScreen/index.js

```diff
@@ -1,8 +1,9 @@
 import React, { useRef, useState, useCallback } from 'react';
```

Comment:
This file is too long. I think you can move all the create new target parts to a new `CreateTaget` component and then import the component here. WDYT?